lightwood/api/json_ai.py

```diff
@@ -578,6 +578,11 @@ def _add_implicit_values(json_ai: JsonAI) -> JsonAI:
         if json_ai.features[name].dependency is None:
             json_ai.features[name].dependency = []
 
+    imputers = {}
+    for col, feat in json_ai.features.items():
+        if feat.imputer is not None:
```

Comment:
the approach is good but some imputers may require dependency info from other columns, which is not currently accounted for in json-ai